README.md

```diff
@@ -111,6 +111,15 @@ pressing `C-c C-r`.
   recover the original behavior by customizing this user option; see
   the docstring for more details.
 
+* `ivy-prescient-override-filter-method`: If non-nil, override the
+  default in `ivy-re-builder-alist` with `prescient-filter-method`.
+  You can set this to nil to retain filter functions set by ivy.
+
+* `ivy-prescient-adaptive-sorting`: If non-nil, use adapative sorting
+  for ivy collections. This will override the default in
+  `ivy-sort-functions-alist`. YOu can set this to nil to retain
+  default ivy sorting.
+
```

Comment:
These variable names seem hard to remember to me. The rest of the documentation makes it clear that `prescient.el` does two things: sorting and filtering. What about naming the variables `ivy-prescient-enable-sorting` and `ivy-prescient-enable-filtering`, instead?

Then the descriptions seem naturally simplified to e.g. "If set to nil, then `ivy-prescient.el` does not change the sorting behavior of Ivy from the default. See the Ivy documentation for information about how to configure Ivy's sorting yourself." It doesn't seem necessary to me to mention specific variables from Ivy and how they are affected, as that's an implementation detail. (The user should either read all about this in the Ivy documentation, or be completely unaware of it and just use the `prescient.el` interface.)